Dockerfile

```diff
@@ -11,7 +11,7 @@ WORKDIR /app
 # This will allow Docker to reuse a cached image if no dependencies have changed
```

Comment:
Reverted a temporary removal of package-lock from the dockerfile process due to constant changes. Thanks for remembering to put this back.